src/app/components/home/partners/partners.component.html

```diff
@@ -32,6 +32,9 @@
 			<td>
 				<img class ="partner_img" src = "assets/images/partners/virginiatech.png"/>
 			</td>
+			<td>
+				<img class ="partner_img" src = "assets/images/partners/argoai.png"/>
+			</td>
 		</tr>
 	</table></div>
 </div>
```

Comment:
Please add a new line here.

Reply:
> Please add a new line here.

Hey, I'm not sure what you mean by a new line

Reply:
@rlee7 can you add a new line after last `</div>`